src/views/Settings/Taxes.tsx

```diff
@@ -0,0 +1,14 @@
+import { Heading, Link } from '@chakra-ui/react';
+import { Link as RouterLink } from 'react-router-dom';
+import { ROUTE_SETTINGS_FORMS } from 'routing';
+
+const Taxes = () => (
```

Comment:
It would be nice to display table with taxes as you already saving those documents in Firebase.
You can rely on other components like `Categories`.

Reply:
Ok, I've added a separate ticket in Jira backlog (MP-92) to not block progress within this PR.